install.js

```diff
@@ -18,7 +18,10 @@ function xmkdirSync (path, mode) {
       console.error('Make sure you have the necessary permissions')
       process.exit(1)
     } else if (/EEXIST/.test(err.message)) {
-      // do nothing
+      // do nothing, but this might be a file, not a directory
+    } else if (/ENOTDIR/.test(err.message)) {
+      console.error('Will not install in a git submodule')
```

Comment:
Since we don't consider this an error, should use `console.log` here. Otherwise looks good!